public/styles.css

```diff
@@ -236,6 +236,10 @@ nav.sidebar {
 }
 
 .full-screen-overlay {
+    z-index: 9998;
```

Comment:
Are you sure this is needed? It shouldn't be.
z-index is relevant only for relative and absolutely positioned things The content of full-screen-overlay are not using this, so they do not need to be z-indexed.